rcl/test/rcl/test_wait.cpp

```diff
@@ -179,11 +181,11 @@ TEST(CLASSNAME(WaitSetTestFixture, RMW_IMPLEMENTATION), multi_wait_set_threaded)
             // printf("%s\n", ss.str().c_str());
           }
           ret = rcl_wait(&test_set.wait_set, wait_period);
+          ASSERT_NE(ret, RCL_RET_ERROR);
           if (ret != RCL_RET_TIMEOUT) {
             {
               std::stringstream ss;
-              ss << "[thread " << test_set.thread_id << "] Wakeup  (try #" << wake_try_count << ")";
-              // printf("%s\n", ss.str().c_str());
+              ss << "[thread " << test_set.thread_id << "] Wakeup (try #" << wake_try_count << ")";
```

Comment:
I left the print statements commented out in case they're helpful for debugging later. Can you either put the printf back as a comment or remove all of them? Either is fine by me. (also the extra whitespace was so the output of `(try ..` lined up)